habitat_baselines/evaluate_ppo.py

```diff
@@ -149,13 +212,71 @@ def main():
         episode_counts += 1 - not_done_masks
         current_episode_reward *= not_done_masks
 
+        # added for video support
+        next_episodes = envs.current_episodes()
+        envs_to_pause = []
+        n_envs = envs.num_envs
+        for i in range(n_envs):
+            if next_episodes[i].episode_id in stats_episodes:
+                envs_to_pause.append(i)
+
+            if not_done_masks[i].item() == 0:
+                # episode ended, record and generate videos
+                stats_episodes.add(current_episodes[i].episode_id)
+                if args.video == 1 and len(rgb_frames[i]) > 0:
+                    video_name = "episode{}_ckpt{}_spl{:.2f}".format(
+                        current_episodes[i].episode_id,
+                        checkpoint_idx,
+                        infos[i]["spl"],
+                    )
+                    images_to_video(
+                        rgb_frames[i], args.out_dir_video, video_name
+                    )
+                    frames_to_tb_video(
```

Comment:
Plan to make video option [0, 1, 2, 3] instead of [0, 1]. 0 for no video, 1 for saving to dir, 2 for tb, 3 for both, which avoids making a separate flag.

Reply:
Maybe named options make more sense? i.e. `["no_video", "video_dir", "video_tb", "video_both"]`?